aries-site/src/examples/templates/dashboards/content/chartCards/RulesAudit.js

```diff
@@ -0,0 +1,191 @@
+        if (!nextFrameworks.includes(check.frameworkId)) {
+          nextFrameworks.push(check.frameworkId);
+        }
+      });
+      setRules(nextRules);
+      setFrameworks(nextFrameworks);
+    }
+  }, [checks, completedAudits]);
+
+  // Assemble chart data
+  useEffect(() => {
+    const nextAuditResults = [
+      { result: 'pass', count: 0, percent: 0, color: 'status-ok' },
+      { result: 'fail', count: 0, percent: 0, color: 'status-critical' },
+      { result: 'error', count: 0, percent: 0, color: 'status-warning' },
+    ];
+    const ruleMap = new Map();
+    nextAuditResults.forEach(result => {
+      const index = nextAuditResults.findIndex(
+        el => el.result === result.result,
+      );
+      ruleMap.set(result.result, index);
+    });
+    rules.forEach(rule => {
+      const index = ruleMap.get(rule.result);
+      nextAuditResults[index] = {
+        ...nextAuditResults[index],
+        count: nextAuditResults[index].count + 1,
+        percent: ((nextAuditResults[index].count + 1) / rules.length) * 100,
+      };
+    });
+    setAuditResults(nextAuditResults);
+  }, [rules]);
+
+  const grid = {
+    columns: ['auto', 'auto', 'auto'],
+    rows: ['auto', 'auto'],
+    areas: [
+      ['rules', 'frameworks', 'audits'],
+      ['chart', 'chart', 'chart'],
+    ],
+  };
+
+  const nextScheduledAudit = nextAudit && (
+    <Box direction="row" gap="xsmall">
+      <Box
+        // pad aligns icon and text first row
+        pad={{ top: 'xsmall' }}
+      >
+        <Calendar size="small" color="text-strong" />
+      </Box>
+      <Text size="small" weight="bold">
+        Next audit will be performed in{' '}
+        {Math.round(
+          (new Date(nextAudit.scheduledDate) - new Date()) /
+            (1000 * 60 * 60 * 24), // milliseconds to days
+        )}{' '}
+        days.
+      </Text>
+    </Box>
+  );
+
+  return (
+    <ChartCard title="Compliance" subtitle={period} footer={nextScheduledAudit}>
+      <Grid
+        columns={grid.columns}
+        rows={grid.rows}
+        areas={grid.areas}
+        gap="medium"
+      >
+        <Measure
+          gridArea="rules"
+          name={{ label: { label: 'Rules', size: 'medium' } }}
+          value={rules?.length}
+        />
+        <Measure
+          gridArea="frameworks"
+          name={{ label: { label: 'Frameworks', size: 'medium' } }}
+          value={frameworks?.length}
+        />
+        <Measure
+          gridArea="audits"
+          name={{ label: { label: 'Audits', size: 'medium' } }}
+          value={completedAudits?.length}
+        />
+        <AuditResultsChart gridArea="chart" data={auditResults} />
+      </Grid>
+    </ChartCard>
+  );
+};
+
+RulesAudit.propTypes = {
+  period: PropTypes.oneOf(['Last 30 Days', 'Last Year', 'Lifetime']),
+};
+
+const AuditResultsChart = ({ data, ...rest }) => (
+  <MeterGroup {...rest}>
+    {data.map(datum => (
+      <Fragment key={datum.result}>
+        <Text>{datum.result}</Text>
```

Comment:
Should we title case?

<img width="143" alt="Screen Shot 2022-02-23 at 8 54 14 AM" src="https://user-images.githubusercontent.com/12522275/155367586-303ac7e3-c5bc-4497-bc0e-35c145c5f498.png">

Reply:
Yes. I missed that when switching from hard coded data to "real".

I will fix.

Reply:
Fixed.